tree/dataframe/test/SimpleFiller.h

```diff
@@ -0,0 +1,26 @@
+#ifndef RDF_TEST_SIMPLEFILLER
+#define RDF_TEST_SIMPLEFILLER
+
+#include <TH1D.h>
+
+class SimpleFiller {
+   TH1D fHisto;
+
+public:
+   SimpleFiller() : fHisto("", "", 128, 0., 0.) {}
+   SimpleFiller(const SimpleFiller &) = default;
+   SimpleFiller(SimpleFiller &&) = default;
```

Comment:
I am never 100% sure what constructors the compiler generates, I prefer explicitly asking for them so it complains if at some point they can't be generated rather than silently stop generating them.